modules/chatops4s-slack/src/main/scala/chatops4s/slack/SlackClient.scala

```diff
@@ -0,0 +1,40 @@
+package chatops4s.slack
+
+import cats.effect.IO
+import cats.implicits.*
+import chatops4s.slack.models.*
+import chatops4s.slack.models.SlackModels.given
+import io.circe.syntax.*
+import sttp.client4.*
+import sttp.client4.cats.CatsBackend
+import sttp.client4.circe.*
+
+class SlackClient(config: SlackConfig, backend: CatsBackend[IO]) {
```

Comment:
Dont commit on a backend here. You should use `just `Backend`. You can keep `IO` fixed for now